engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/settings/SettingsMenuScreen.java

```diff
@@ -28,9 +28,7 @@ public void initialise() {
         WidgetUtil.trySubscribe(this, "video", button -> triggerForwardAnimation(VideoSettingsScreen.ASSET_URI));
         WidgetUtil.trySubscribe(this, "audio", button -> triggerForwardAnimation(AudioSettingsScreen.ASSET_URI));
         WidgetUtil.trySubscribe(this, "input", button -> triggerForwardAnimation(InputSettingsScreen.ASSET_URI));
-        WidgetUtil.trySubscribe(this, "test", button -> triggerForwardAnimation(new ResourceUrn("engine:autoConfigTestScreen")));
-        WidgetUtil.trySubscribe(this, "test2", button -> triggerForwardAnimation(new ResourceUrn("engine:mathTypeWidgetTestScreen")));
-        WidgetUtil.trySubscribe(this, "test3", button -> triggerForwardAnimation(new ResourceUrn("engine:builtinTypeWidgetTestScreen")));
+        WidgetUtil.trySubscribe(this, "auto", button -> triggerForwardAnimation(new ResourceUrn("engine:autoConfigTestScreen")));
```

Comment:
Are we still keeping this here? If yes, we must make sure to remove this before the next release. I would still prefer getting rid of this button entirely and using the `showScreen engine:autoConfigTestScreen` command to load the screen that displays test `AutoConfig`s whenever needed for testing. That's the approach I used to load the type widget factory test screens as well.

Reply:
Oh. I forgot to refactor names.

It should be AutoSettingsScreen. Not Test.

It should be as place, which can be improved soon. Or contract place between autoconfigs and typewidgetlibrary

Reply:
Ah, so is it the screen that currently houses all `AutoConfig`s (like `AudioConfig` and in the future `SystemConfig`? If it is a non-testing screen and is actually meant to be exposed to players, then we can keep this here. We should in the near future figure out a good way to expose autoconfig screens to the end user though.